sparkle/src/components/CardButton.tsx

```diff
@@ -56,10 +31,21 @@ const cardButtonVariants = cva(
   }
 );
 
+interface CardButtonProps
+  extends React.ButtonHTMLAttributes<HTMLButtonElement>,
+    VariantProps<typeof cardButtonVariants> {
+  children: ReactNode;
+  href?: string | UrlObject;
+  target?: string;
+  rel?: string;
+  replace?: boolean;
+  shallow?: boolean;
+}
+
 export function CardButton({
   children,
-  variant,
-  size,
+  variant = "primary",
```

Comment:
If you add a "defaultVariants" in cva you do not need to set a default value in the props 😃